packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -554,15 +554,17 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     });
   }
 
-  public joinCall(microphoneOn?: boolean): CallTypeOf<AgentType> | undefined {
+  public joinCall(microphoneOn?: boolean, cameraOn?: boolean): CallTypeOf<AgentType> | undefined {
     if (_isInCall(this.getState().call?.state ?? 'None')) {
       throw new Error('You are already in the call!');
     }
 
     return this.teeErrorToEventEmitter(() => {
       const audioOptions: AudioOptions = { muted: !(microphoneOn ?? this.getState().isLocalPreviewMicrophoneEnabled) };
       // TODO: find a way to expose stream to here
-      const videoOptions = { localVideoStreams: this.localStream ? [this.localStream] : undefined };
+      const selectedCamera = getSelectedCameraFromAdapterState(this.getState());
+      const localStream = selectedCamera ? new SDKLocalVideoStream(selectedCamera) : undefined;
+      const videoOptions = localStream && cameraOn ? { localVideoStreams: [localStream] } : {};
```

Comment:
This is a breaking change.

If a customers code turns on the camera then they have:
```
joinCall(true);
```
They want to join the call with cameraOn. However, with this change we will now interpret this as:
```
joinCall(true, false);
```
Which is not their intention.

Perhaps the parameter of this function should be `camera?: true | false | 'auto'` where auto is the default?
Maybe another argument for creating a new function to handle this called something like joinCallWithOptions

Reply:
If customer call joinCall(true), they will only join call with mic on.
And calling joinCall() means joining with mic off and camera off

Reply:
Today though, if a customer calls `joinCall(true)` they join the call with micOn, and with camera on if this.localStream is true

Reply:
sorry I don't follow why this is breaking @JamesBurnside are you trying to say that if you have code to turn on the camera in contoso's application that setting the second value as false will break that on skip screens?